common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -159,6 +166,8 @@ def test_full_name_field(self):
             [u'another name', self.USERNAME],
         )
 
+        self.assert_event_emitted_num_times('edx.user.settings.changed', self.start_time, self.user_id, 2)
```

Comment:
One event for each new valid value in `[u'another name', self.USERNAME]`.

The unit tests currently verify data in the event, so I don't see a need to duplicate that here.